server/routerlicious/packages/routerlicious-base/src/utils/constants.ts

```diff
@@ -8,5 +8,11 @@ export const Constants = Object.freeze({
 	createDocThrottleIdPrefix: "createDoc",
 	getDeltasThrottleIdPrefix: "getDeltas",
 	getSessionThrottleIdPrefix: "getSession",
+	socketConnectionsThrottleIdPrefix: "socketConnections",
+	submitOpsThrottleIdPrefix: "submitOps",
+	submitSignalThrottleIdPrefix: "submitSignal",
 	generalRestCallThrottleIdPrefix: "generalRestCall",
+	throttleGeneralTenant: "generalTenant",
+	throttleGeneralCluster: "generalCluster",
+	throttleTenantGroup1: "tenantGroup1",
```

Comment:
The last three are throttler names and Not prefixes, right?
 
If yes, then maybe we should name them differently. constant variable name could be something like:
generalTenantThrottler: "generalTenant",
generalClusterThrottler: "generalCluster",
tenantGroup1Throttler: "tenantGroup1",


I do feel this list is growing and we will need to refactor a lot of this when we have some more bandwidth :(

Reply:
Sounds good! Done. :)